packages/media_core/src/endpoint.rs

```diff
@@ -113,6 +131,7 @@ pub enum EndpointReq {
     AudioMixer(EndpointAudioMixerReq),
     RemoteTrack(RemoteTrackId, EndpointRemoteTrackReq),
     LocalTrack(LocalTrackId, EndpointLocalTrackReq),
+    MessageChannel(String, EndpointMessageChannelReq),
```

Comment:
We should create MessageChannelId(String) or MessageChannelLabel(String) for avoiding wrong with other string